client/object_put.go

```diff
@@ -113,6 +113,10 @@ func (x *PrmObjectPutInit) MarkLocal() {
 // writeHeader writes header of the object. Result means success.
 // Failure reason can be received via [DefaultObjectWriter.Close].
 func (x *DefaultObjectWriter) writeHeader(hdr object.Object, copyNum uint32) error {
+	if x.opts.session != nil && x.opts.sessionV2 != nil {
```

Comment:
Not sure if it's possible, tokens are probably checked already by the time we get `DefaultObjectWriter`. Only `ObjectPutInit()` creates it.